torch/distributions/von_mises.py

```diff
@@ -51,7 +51,6 @@ def _log_modified_bessel_fn(x, order=0):
     return result
 
 
-@torch.jit.script
 def _rejection_sample(loc, concentration, proposal_r, x):

```

Comment:
Probably should be removed, i think there was a reason stated in the PR for why this was done.